docs/source/theory/intro.rst

```diff
@@ -48,6 +48,19 @@ This is an overview of physical assumptions implemented in the numerics of Impac
 Tracking and Lattice Optics
 """""""""""""""""""""""""""
 
+Tracking through lattice optics in ImpactX is performed by updating the canonical phase space variables (x,px,y,py,t,pt) using symplectic transport.
+The elements supported currently fall into one of three categories:
+
+* **ideal (thick) elements** using a hard-edge fringe field approximation, such as drifts, quadrupoles, and dipoles
+* **zero-length (thin) elements**, such as multipole kicks and coordinate transformations
+* **soft-edge elements** described by :math:`s`-dependent user-provided field data
+
+Transport may be performed using one of three possible levels of approximation to the underlying Hamiltonian:
+
+* **linear transfer map (default):** obtained by expanding the Hamiltonian through terms of degree 2 in the deviation of phase space variables from those of the reference particle
+* **chromatic or paraxial approximation:** obtained by expanding the Hamiltonian through terms of degree 2 in the transverse phase space variables, while retaining the nonlinear dependence on the energy variable pt
+* **exact Hamiltonian:** obtained using the exact nonlinear Hamiltonian
+
 * **tracking through lattice optics:** is treated through linear order with respect to the reference particle
 
   * **velocity spread:** the above linearization implies that, when solving space-charge effects, we assume that the relative spread of velocities of particles in the beam is negligible compared to the velocity of the reference particle
```

Comment:
Formatting issue with the list above now:
```suggestion
The **tracking through lattice optics** is treated through linear order with respect to the reference particle

* **velocity spread:** the above linearization implies that, when solving space-charge effects, we assume that the relative spread of velocities of particles in the beam is negligible compared to the velocity of the reference particle.
```

I have to admit that this last paragraph now reads a bit confusing compared to the chromatic details in the new paragraphs above it.

Reply:
I suggest deleting the bullet about tracking through linear order, since this is not always true, and it has been replaced by the comments above about levels of approximation.  What do you think?

The comment about velocity spread could be moved to the section on space charge, and we could take out the phrase "the above linearization implies that".

Reply:
Yes, good idea to be removing and moving.

Reply:
Done.